### ISSUE TITLE
GANN for Pascal-VOC

### ISSUE THREAD

**Issue by nurlanov-zh** [Thinklab-SJTU/ThinkMatch#30]
Hello, 
I am wondering if the results of GANN on Pascal-VOC were published? I have seen the experimental yaml as well as the pretrained weights, but couldn't find it in the publications.
Also the question regarding evaluation: according to the code you validate (during training) and test (after training for comparison of models) on the same data, is it correct? -- the 'test' set of each dataset. Do the numbers in the tables correspond to such a protocol?

Thanks for such a nice library! 
I hope to get published with some follow-ups and I will definitely cite your works.

**Comment by rogerwwww** [comment]
Hi there,

The results of GANN on Pascal-VOC are under submission so it is not published. It seems to be a mistake by us to release part of the code of an unpublished paper and I feel sorry about that. Perhaps it will be better to refer to the results of GANN on other datasets (where the results are already published and cross-checked). Currently, we are working on a benchmark and I hope it will be published in the coming several months.

For the train/test split, I guess you are mentioning the train/val/test split for the dataset? We follow the train/test split by the first deep graph matching paper (Zanfir, CVPR 2018) where there is no validation set involved. Also, it seems that all existing methods follow such a protocol.

Many thanks for your interest in our work!

**Comment by rogerwwww** [comment]
Hi, I am closing this issue since there are no follow-ups. If anyone has any questions, just feel free to reopen it.
